README.md

```diff
@@ -50,6 +50,11 @@ Running the tests necessary to merge into the repository requires:
  * Python 3.7.x (not 3.8 or later), and
  * [PyPy 7.3.x](https://www.pypy.org/).
 
+`eth1.0-specs` depends on a submodule that contains common tests that are run across all clients, so we need to clone the repo with the --recursive flag. Example:
+```bash
+$ git clone --recursive https://github.com/quilt/eth1.0-specs.git
+```
+
```

Comment:
Would you mind bumping this above "Running the tests necessary..."? It seems out of place between the required versions, and the note about why those versions are required.

Reply:
Oh! And could you add something like:

> Or, if you've already cloned the repository, you can fetch the submodules with:
>
> ```bash
> $ git submodule update --init --recursive
> ```

Thanks a ton!

Reply:
> Would you mind bumping this above "Running the tests necessary..."? 

Nice catch!
> Or, if you've already cloned the repository, you can fetch the submodules with:

Done.